docs/topics/OAuth2.md

```diff
@@ -19,7 +19,7 @@ The first step in implementing OAuth2 is [registering a developer application](#
 
 ###### OAuth2 Scopes
 
-These are a list of all the OAuth2 scopes that Discord supports. Scopes that are behind a whitelist cannot be requested unless your application is on said whitelist, and may cause undocumented/error behavior in the OAuth2 flow if you request them from a user.
+These are a list of all the OAuth2 scopes that Discord supports. Scopes that are behind an allowlist cannot be requested unless your application is on said allowlist, and may cause undocumented/error behavior in the OAuth2 flow if you request them from a user.
```

Comment:
"require approval" and "approval list" maybe?

Reply:
Yeah I think this is a good point. I don't think we should just s/whitelist/allowlist. There are other ways of saying this. I agree with moving away from the term whitelist, I just don't like how "allowlist" sounds 🤣 

Saying "scopes that require approval" for example is a much better way of writing it